src/Umbraco.Web/MembershipProviderExtensions.cs

```diff
@@ -23,7 +20,7 @@ internal static class MembershipProviderExtensions
             return new Dictionary<string, object>
                 {
                     {"minPasswordLength", membershipProvider.MinRequiredPasswordLength},
-                    {"enableReset", membershipProvider.EnablePasswordReset},
+                    {"enableReset", UmbracoContext.Current.Security.CurrentUser.IsAdmin()},

```

Comment:
This is incorrect though, if the user is not an admin than this setting membershipProvider.EnablePasswordReset should still be used.